app/frontends/ogc-coverages/get-coverage-rangeset.ts

```diff
@@ -70,6 +70,12 @@ export default function getCoverageRangeset(
     if (bbox) {
       operation.boundingRectangle = bbox;
     }
+    const point = parsePointParam(query.point);
+    if (point) {
+      if (bbox || req.rawHeaders.filter((x) => new RegExp('shapefile=*').test(x)).length)
```

Comment:
Ideally we should be able to check for `req.context.shapefile`, and if not that `req.operation.geojson` instead of a regex on the headers. If neither of those work I'm ok with merging this in.

Reply:
Both `req.context.shapefile` and `req.operation.geojson` is not generated at this point.